src/index.js

```diff
@@ -363,6 +421,26 @@ class SoshContentScript extends ContentScript {
     )
   }
 
+  async waitForUserAction(url) {
+    this.log('info', 'waitForUserAction start')
+    await this.setWorkerState({ visible: true, url })
+    await this.runInWorkerUntilTrue({ method: 'waitForCaptchaResolution' })
+    await this.setWorkerState({ visible: false, url })
+  }
+
+  async getUserDataFromWebsite() {
+    this.log('info', '🤖 getUserDataFromWebsite starts')
+    await this.waitForElementInWorker('.dashboardConso__welcome')
+    const sourceAccountId = await this.runInWorker('getUserMail')
+    if (sourceAccountId === 'UNKNOWN_ERROR') {
```

Comment:
Maybe we can avoid this kind of condition ? 

IMO it would fit better with the actual conventions ?

Reply:
Fixed